src/Controller/CrudController.php

```diff
@@ -171,14 +172,22 @@ public function editAction(Request $request): Response
 
         $form = $view->getEditForm();
 
-        $form->handleRequest($request);
+        try {
+            $form->handleRequest($request);
 
-        if ($form->isSubmitted()) {
-            $this->dispatchEvent(CrudEvent::PRE_VALIDATE_PREFIX, $entity);
-            if ($form->isValid()) {
-                return $this->formSubmittedAndValid($entity, $mode, Page::EDIT);
+            if ($form->isSubmitted()) {
+                $this->dispatchEvent(CrudEvent::PRE_VALIDATE_PREFIX, $entity);
+                if ($form->isValid()) {
+                    return $this->formSubmittedAndValid($entity, $mode, Page::EDIT);
+                }
+                throw new FlashBagExecption('error', 'araise_crud.save_error');
             }
-            $this->addFlash('error', 'araise_crud.save_error');
+        } catch (FlashBagExecption $e) {
+            $this->addFlash($e->getFlashType(), $e->getFlashMessage());
+            $this->container->get(LoggerInterface::class)->warning('Error while editing: '.$e->getMessage(), [
```

Comment:
this is misleading. `warning: "Error while ...` As the FlashBagException will only be thrown for Errors users should see, There is no need to log here! (this for all cases)

Reply:
@tuxes3 I guess that statement is valid for all methods where I used the FlashBagException. I will change them accordingly 👍